src/tabs/osd.html

```diff
@@ -122,15 +122,26 @@ <h1 class="tab_title">
                         <h1 class="tab_title" i18n="osdSetupFontPresets" />
                         <div class="content_wrapper font-preview"></div>
                         <div class="fontbuttons">
-                            
+
+                        </div>
+                        <h1 class="tab_title" i18n="osdSetupLogo" />
+                        <div class="helpicon cf_tip" i18n_title="osdSetupReplaceLogoHelp"></div>                                                    
+                        <div class="content_wrapper" style="background:#00dd00; line-height: 0; margin-bottom:10px">
```

Comment:
I know this is being done in other places, but is better if we try to avoid the use of the style attribute. Is far better to define in css and is a way to identify similar styles and simplify the html code.

Reply:
The idea is that any new code added, be better than it is.

The CSS gives a lot of power, the idea is to have "general styles" as css classes, but you can, in the CSS file for this HTML, modify the behaviour for things like "the class with name X" or "all the div that are inside the name/class Y" or "all the X class inside the Y name".

The developers tool of chrome gives a lot of help in future updates and maintenance to deal with this. 

If this must be part of this PR or not I can't say.

Reply:
I would like to see it as part of this pull request. It's too late to get this translated in time for the next configurator release (early next week), so we may as well spend the time to add css styles here, and leave the conversion of the rest of this tab for a future pull request.